src/app/entities/courses/load-more-panel/load-more-panel.component.spec.ts

```diff
@@ -22,4 +23,13 @@ describe('LoadMorePanelComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should emit `loadMore` event when click on `Load more` button', () => {
+    const onLoadMoreEmitSpy = spyOn(component.loadMore, 'emit');
+
+    const loadMoreButton = fixture.debugElement.query(By.css('.button.button--link'))
```

Comment:
you can add shared utility to DRY:
`getFixtureDebugElementBySelector(fixture, cssSelector)`

Reply:
Will do, thanks